### DISCUSSION TITLE
Error 500 for "Add BREAD to table"

### DISCUSSION THREAD

**Discussion by phenomenia** [thedevdojo/wave#121] [Q&A] [answered]
When I want to add BREAD to a table in the admin area I get an error 500 when I submit the form.
This is for new table as wenn as for existing ones (to test with), e.g. the permission_groups table.

How can this be fixed?

**Comment by bobbyiliev** [comment, marked as answer]
Hi,

I just tested this and it seems to be all working as expected. Did you also create a valid model when you created your table?

Also, the 500 error is just a generic error, whenever you get such an error during development, it is best to enable debugging so that you could see the actual PHP error that is causing the issue rather than the generic 500 error. That way you will be able to tell what actually is going on and start from there.

Once you've enabled debugging feel free to share the actual error here.
